src/components/dataPanel/FieldValue.test.tsx

```diff
@@ -11,8 +11,14 @@ describe("FieldValue", () => {
   it("renders string values in bold", () => {
     render(<FieldValue>Actual beacon data field</FieldValue>);
 
-    expect(screen.getByText("Actual beacon data field")).toHaveStyle(
+    expect(screen.getByText(/Actual beacon data field/i)).toHaveStyle(
```

Comment:
Makes `getByText` case insensitive, so that the test is only asserting on the `font-weight`.